README.md

```diff
@@ -1,2 +1,157 @@
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Math Magician] <a name="about-project"></a>
+
+**Math Magician** is an application that allows users to make simple calculations.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+    <li>Webpack</li>
+    <li>Jest</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Created with react**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo">Comming soon...</a>
+
+<!-- - <a href="https://demesameneshoa.github.io/JavaScript-Capstone/">Click Here</a> to see the live demo -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites <a name="prerequisites">
+
+In order to run this project you need:
+
+- Visual Studio Code
+- Node JS
+
+### Setup <a name="setup">
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  mkdir math-magician
+  cd math-magician
+  git clone https://github.com/Gh-Reza/math-magicians.git
+```
+
+### Install <a name="install">
+
+```
+npm install
+npm install jest-dom
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage <a name="install">
+
+Initialize it with:
+
+```
+npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Reza Merzaie**
+
+- GitHub: [@Gh-Reza](https://github.com/Gh-Reza)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add calculation function**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please show us support by givintg it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thanks Microverse for this learning opportunity.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.

```

Comment:
- [ ] MIT License link is broken, kindly ensure to fix it.

Reply:
- [ ] It would be better to fix the broken link as mentioned earlier. Quick fix: kindly change `./MIT.md` to `./License` because that is the name of the license file you created